website/docs/cloud-docs/api-docs/data-retention-policies.mdx

```diff
@@ -0,0 +1,301 @@
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/workspaces/ws-UYv6RYM8fVhzeGG5/relationships/data-retention-policy
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "drp-Kh74zpKVGNWbpugQ",
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "delete-older-than-n-days": 33
+    },
+    "relationships": {
+      "target": {
+        "data": {
+          "id": "ws-7aiqKYf6ejMFdtWS",
+          "type": "workspaces"
+        }
+      }
+    }
+  }
+}
+```
+
+## Remove Data Retention Policy
+
+This endpoint removes the data retention policy explicitly set on a target resource.
+
+| Resource      | API Endpoint                                    | 
+| --------------| ----------------------------------------------- |
+| [Workspaces](/terraform/cloud-docs/api-docs/workspaces#remove-data-retention-policy)       | `DELETE /workspaces/:workspace_id/relationships/data-retention-policy`         |
+| [Organizations](/terraform/cloud-docs/api-docs/organizations#remove-data-retention-policy) | `DELETE /organizations/:organization_name/relationships/data-retention-policy` |
+
+| Status  | Response                  | Reason(s)                                                             |
+| ------- | ------------------------- | --------------------------------------------------------------------- |
+| [204][] | No Content                | Successfully removed the target resource's data retention policy.     |
+| [404][] | [JSON API error object][] | Target resource not found, or user unauthorized to perform action.    |
+
+### Request Body
+
+No request body.
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE \
+  https://app.terraform.io/api/v2/organizations/hashicorp/relationships/data-retention-policy
+```
+
+### Response
+
+No response body.
+
+Status code `204`.
+
+
+# Types of Data Retention Policies
+
+Data retention policy APIs accept or return different "types" of data retention policies. Different types of data retention policies have different attributes, and configure how the related resource will retain backing data differently.
+
+When viewing or updating policies through the API, it is important to pay attention to the `"type"` field of the [JSONAPI Resource Object](https://jsonapi.org/format/#document-resource-objects). Most `GET` endpoints are polymorphic and may return different policy types depending on how Terraform Enterprise has been configured. Most `POST` endpoint can be used to change the type of the policy, and different policy types will require different attributes to be set.
+
+Unless otherwise specified, `GET` API endpoints can return any of the following data retention policy types, and `POST`/`PATCH` endpoints can be used to set or update the data retention policy to any of the following types.
```

Comment:
Non-blocking suggestion (same as above!): Should we replace this period with a colon since we're about to list out the policy types?
```suggestion
Unless otherwise specified, `GET` API endpoints can return any of the following data retention policy types, and `POST`/`PATCH` endpoints can be used to set or update the data retention policy to any of the following types:
```